AndriyZhornyak/Eveything/HMW11/HMW11.py

```diff
@@ -0,0 +1,40 @@
+# Task1
```

Comment:
def task1():
    try:
        ask = int(input("Enter a number: "))
        if ask <= 0:
            raise ValueError("Number must be positive and greater than 0.")
        if ask % 2 == 0:
            print("The number is even.")
        else:
            print("The number is odd.")
    except ValueError as e:
        print(e)


task1()